oak_utils/src/lib.rs

```diff
@@ -204,3 +204,42 @@ where
         .compile_protos(inputs, includes)
         .expect("could not run prost-build");
 }
+
+/// Options for building gRPC code.
+pub struct CodegenOptions {
```

Comment:
I think I've mentioned this earlier, but I would recommend to add something like a builder, just so you don't need to change example code when you add fields here.

A builder adds a lot of code if you don't want to use a crate like [`derive_builder`](https://docs.rs/derive_builder/0.9.0/derive_builder/), but instead you could get by just by adding a `#[derive(Default)]` on this struct and adding `..Default::default()` in the clients. [Here is an example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f9bbadbc5f3429f2d77167cf3b352c12)

Reply:
Done